src/main/java/com/google/sps/data/Place.java

```diff
@@ -0,0 +1,54 @@
+// Copyright 2019 Google LLC
```

Comment:
That's a great point, I think we will think further about directories when we write the actual project with angular, because everything is going to change